src/box/log.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#pragma once
+
+/**
+ * Check parameters for say_logger_init.
+ *
+ * @retval  0 - Parameters are valid.
+ * @retval -1 - Parameters are not valid (diag is set in this case).
+ */
+int
+say_check_cfg(const char *log,
```

Comment:
This function declaration is redundant - we never call it from C code. Anyway, I don't think that this function deserves a separate file - let's leave it in `box.cc`, where we have all other config-checking functions? Or maybe move it to `src/lib/core/say.c` and return config name + error string instead of `ER_CFG`? (The core lib can't use `ClientError`.) Or maybe rewrite it completely in Lua?

Reply:
As discussed offline - leave it in box.cc.